### ISSUE TITLE
Mute from sidebar as moderator

### ISSUE THREAD

**Issue by PVince81** [nextcloud/spreed#4412]
**Is your feature request related to a problem? Please describe.**
When the list of participants is very long, it would be convenient to be able to simply search for a participant in the sidebar and mute them.
Without this feature, one might need to click the chevron/arrow to iterate over all the participants one by one and see who is not muted, which is tedious and a loss of time.

**Describe the solution you'd like**
1. The participants sidebar already displays microphones but it should also show muted state. This would make it possible to find out who is not muted. Everyone should see this.
2. Allow moderators to click on a microphone icon to mute or unmute that participant.

**Describe alternatives you've considered**

**Additional context**
See existing microphone button.

**Comment by danxuliu** [comment]
To be able to implement this first there should be a way to associate the Nextcloud session ID with the external signaling session ID, which as far as I know is not possible yet (but maybe it will come with the SIP changes, I do not know).

**Comment by PVince81** [comment]
At least so far it seems the sidebar is already aware about who has audio. Unless the microphone only means they are inside the meeting

**Comment by nickvergessen** [comment]
> Unless the microphone only means they are inside the meeting

Exactly. It's only about their "in_call" flag in the participant list and that currently doesn't update.
But with the SIP changes we are coming a bit closer to this I think.

**Comment by PVince81** [comment]
Also:
- display "muted" vs "muted" icon to make it more useful to find out who forgot to mute, and then click to mute that participant
